k8s/helm chart/argilla/templates/argilla-server-hpa.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: autoscaling/v2

```

Comment:
Suggestion: wrap HPAs in something like `{{- if .Values.autoscaling.enabled }}`

Reply:
https://github.com/bikash119/argilla/blob/70059f21bfd977ee726978ba74f44a4d3d026767/examples/deployments/k8s/argilla-chart/templates/ingress.yaml#L1